### ISSUE TITLE
Support an array of formats in util.styleText

### ISSUE THREAD

**Issue by infrahead** [oven-sh/bun#11705]
### What is the problem this feature would solve?

Was glad to see styleText implemented in #9287 but then shortly thereafter Node added the ability to specify multiple formats in an array.  Was hesitant to mark this as a bug vs feature request.

### What is the feature you are proposing to solve the problem?

https://nodejs.org/api/util.html#utilstyletextformat-text
https://github.com/nodejs/node/pull/52040

### What alternatives have you considered?

continuing to use original API with nested function calls

**Comment by pavi2410** [comment]
Just stumbled upon this issue

**Comment by aboqasem** [comment]
```
❯ node -e 'import util from "node:util"; console.log(util.styleText(["green", "bgBlack"], "hello"))';
hello
❯ bun -e 'import util from "node:util"; console.log(util.styleText(["green", "bgBlack"], "hello"))';
1 | import util from "node:util"; console.log(util.styleText(["green", "bgBlack"], "hello"))
                                                   ^
error: The value "green,bgBlack" is invalid for argument 'format'. Reason: must be one of: reset, bold, dim, italic, underline, blink, inverse, hidden, strikethrough, doubleunderline, black, red, green, yellow, blue, magenta, cyan, white, bgBlack, bgRed, bgGreen, bgYellow, bgBlue, bgMagenta, bgCyan, bgWhite, framed, overlined, gray, redBright, greenBright, yellowBright, blueBright, magentaBright, cyanBright, whiteBright, bgGray, bgRedBright, bgGreenBright, bgYellowBright, bgBlueBright, bgMagentaBright, bgCyanBright, bgWhiteBright
 code: "ERR_INVALID_ARG_VALUE"

      at styleText (node:util:131:19)
      at /Users/zouabi/dev/dotfiles/[eval]:1:48

Bun v1.1.33 (macOS x64)
```

**Comment by pavi2410** [comment]
I have a PR already https://github.com/oven-sh/bun/pull/13807

**Comment by infrahead** [comment]
Looks like this shipped in 1.1.41
https://bun.sh/blog/bun-v1.1.41#fixed-util-styletext-supports-color-arrays
